src/content/blog/2023-09-20-tusd-200.md

```diff
@@ -0,0 +1,65 @@
+- **Support for the new IETF protocol:** We are working with the HTTP working group of the IETF on a [new standard for resumable file uploads](/blog/2023/08/09/resumable-uploads-ietf). This release includes preliminary support for this experimental protocol via the `-enable-experimental-protocol` flag.
+- **Updated dependencies:** As with every release, we upgraded all dependencies to their latest versions to ensure that we include all available stability and security fixes.
+
+## Breaking Changes
+
+Unfortunately, this release also brings some breaking changes along. Affected users are required to adjust their tusd installation, even though the necessary changes should be minimal.
+
+- **Require Go 1.20:** Our upgrade policy dictates that we always support the last two major releases of Go. As of the release of tusd v2, this is Go 1.20 and 1.21. Earlier versions are not supported anymore. If you are building tusd on your own or are using it as a package, please ensure that you are using one of these versions.
+- **For users of tusd as a CLI:**
+    - Flags that specify a duration have a new syntax: Before v2, their value was parsed as the number of milliseconds (e.g. `-duration=6000`). Now, the value is parsed as a number with an attached unit of time (e.g. `-duration=6s`).
+    - The `-timeout` flag has been renamed to `-network-timeout` to better describe its actual effect.
+    - As mentioned above, the structure of the data provided in hooks has slightly changed. Please consult the [hook system documentation](https://github.com/tus/tusd/blob/main/docs/hooks.md#hook-requests-and-responses) for the updated structure.
+- **For users of tusd as a package:**
+    - The `Config.DisableCors` field has been removed in favor of the more complete [`Config.Cors` option](https://pkg.go.dev/github.com/tus/tusd/v2/pkg/handler#Config), which allows you to adjust all CORS-related aspects of your handler.
+    - The `s3store` has been upgraded to use the newer AWS SDK for Go v2. Please check the corresponding [migration guide](https://aws.github.io/aws-sdk-go-v2/docs/migrating/) to upgrade from the AWS SDK for Go v1.
+    - The data stores are now required to return an `io.ReadCloser` (instead of just an `io.Reader`) for `GetReader` calls when a client wants to download a file. The additional close functionality is used to better clean up resources.
+
+That being said, tusd v2 does not introduce a breaking change for its HTTP interface. All current tus clients can be used without modifications with tusd v2. Tusd can be upgraded in production from v1 to v2 seamlessly without changes to the clients.
+
+Finally, I want to thank all contributors, who dedicated their time and efforts towards this release! It would not be possible without you!
```

Comment:
Perhaps another section for upcoming future plans? What's cooking for tusd or what are the long-term aspirations?

Reply:
Good idea 👍